features/full_tests/batch_1/identifying_crashes_on_launch.feature

```diff
@@ -0,0 +1,36 @@
+Feature: Identifying crashes on launch
+
+    Scenario: Escaping from a crash loop by reading LastRunInfo in a JVM error
+        When I run "JvmCrashLoopScenario" and relaunch the app
+        And I configure Bugsnag for "JvmCrashLoopScenario"
```

Comment:
I think this step can be removed?  Although it typically goes with the previous, the following step will do the same thing again (I believe).  Same in the NDK scenario.